Airports/EGCC/Sector.txt

```diff
@@ -65,12 +65,12 @@ DISPLAY:CCNTH05:Scottish PC East:CCNTH05
 DISPLAY:CCFIN05:CCFIN05:CCNTH05
 DISPLAY:CCNTH05:CCFIN05:CCNTH05
 COORD:N053.09.54.878:W002.39.12.131
```

Comment:
I realise I've added comments in reverse order on the file... but as below, let's change this so it's on the RMA line:

```suggestion
COORD:N053.10.06.389:W002.38.48.642
```

Reply:
- [ ] Also make sure to change line 60 to read the same, otherwise the lines won't join up.